disperser/dataapi/subgraph_client_test.go

```diff
@@ -411,3 +446,59 @@ func assertGasFees(t *testing.T, gasFees *dataapi.GasFees) {
 	assert.Equal(t, uint64(1000045336), gasFees.GasPrice)
 	assert.Equal(t, uint64(249826325612840), gasFees.TxFee)
 }
+
+func TestQueryOperatorQuorumEvent(t *testing.T) {
+	mockSubgraphApi := &subgraphmock.MockSubgraphApi{}
+	mockSubgraphApi.On("QueryOperatorAddedToQuorum").Return(operatorAddedToQuorum, nil)
+	mockSubgraphApi.On("QueryOperatorRemovedFromQuorum").Return(operatorRemovedFromQuorum, nil)
+	subgraphClient := dataapi.NewSubgraphClient(mockSubgraphApi, &commock.Logger{})
+	result, err := subgraphClient.QueryOperatorQuorumEvent(context.Background(), uint64(78), uint64(88))
+	assert.NoError(t, err)
+
+	addedMap := result.AddedToQuorum
+	assert.Equal(t, 2, len(addedMap))
+	// Quorum events for operator-1.
+	added1, ok := addedMap["operator-1"]
+	assert.True(t, ok)
+	assert.Equal(t, 2, len(added1))
+	assert.Equal(t, "operator-1", added1[0].Operator)
+	assert.Equal(t, uint64(80), added1[0].BlockNumber)
+	assert.Equal(t, 2, len(added1[0].QuorumNumbers))
+	assert.Equal(t, uint8(48), added1[0].QuorumNumbers[0])
```

Comment:
can you explain why this is 48 instead of 01?

Reply:
Added